README.md

```diff
@@ -922,7 +922,10 @@ validator.isInstance(value, target); // Checks value is an instance of the targe
 | `@IsISIN()`                                     | Checks if the string is an ISIN (stock/security identifier).                                                                     |
 | `@IsISO8601()`                                  | Checks if the string is a valid ISO 8601 date.                                                                                   |
 | `@IsJSON()`                                     | Checks if the string is valid JSON.                                                                                              |
-| `@IsLowercase()`                                | Checks if the string is lowercase.                                                                                               |
+| `@IsLowercase()`                                | Checks if the string is lowercase.  
```

Comment:
Formatting. Please do not edit lines out of scope of this PR.

Reply:
This part is still mess up. But clearly you do not know what to do with it. I will fix it later.